### PR TITLE
Scan: Mobile update on block hp

### LINKED ISSUE #2261: Block home page (mobile)
Note that: **No pagination**

### PR DESCRIPTION
**Pull request by pzshine**
Fixed: #2261

## Implementation details

- Increased the spacing on the inner container.
- Implemented the new mobile card component the data tables
  - Now it's only able to use only blockHomepage.
  - This component can be reused.

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: scan/src/pages/BlockHomePage.re [hunk 1]
```diff
@@ -89,13 +89,43 @@ let renderBody = (reserveIndex, blockSub: ApolloHooks.Subscription.variant(Block
...
 
+let renderBodyMobile = (reserveIndex, blockSub: ApolloHooks.Subscription.variant(BlockSub.t)) => {
+  switch (blockSub) {
+  | Data({height, timestamp, validator, txn}) =>
+    <MobileCard
+      values=InfoMobileCard.[
+        ("BLOCK", BlockHeight(height)),
+        ("TIMESTAMP", Timestamp(timestamp)),
+        (
+          "PROPERSER",
```
Comment (tansawit) [inline scan/src/pages/BlockHomePage.re]: "PROPOSER"

File: scan/src/pages/BlockHomePage.re [hunk 2]
```diff
@@ -89,13 +89,43 @@ let renderBody = (reserveIndex, blockSub: ApolloHooks.Subscription.variant(Block
...
+      ]
+      key={height |> ID.Block.toString}
+      idx={height |> ID.Block.toString}
+    />
+  | _ =>
+    <MobileCard
+      values=InfoMobileCard.[
+        ("BLOCK", Loading(70)),
+        ("TIMESTAMP", Loading(166)),
+        ("PROPERSER", Loading(136)),
```
Comment (tansawit) [inline scan/src/pages/BlockHomePage.re]: ditto

File: scan/src/App.re [hunk 3]
```diff
@@ -6,7 +6,7 @@ module Styles = {
   let container =
     style([width(`percent(100.)), height(`percent(100.)), position(`relative)]);
 
-  let innerContainer = style([marginLeft(`auto), marginRight(`auto)]);
+  let innerContainer = style([marginLeft(`auto), marginRight(`auto), paddingLeft(`px(15)), paddingRight(`px(15))]);
```
Comment (evilpeach) [inline scan/src/App.re]: Let's use `padding2`.

File: scan/src/reusable/InfoMobileCard.re [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  | Validator(Address.t, string, string)
+  | Loading(int);
+
+module Styles = {
+  open Css;
+
+  let vFlex = style([display(`flex), alignItems(`center)]);
+  let addressContainer = maxwidth_ => style([alignItems(`center), maxWidth(`px(maxwidth_))]);
+  let datasourcesContainer = style([display(`flex), alignItems(`center), flexWrap(`wrap)]);
+  let oracleScriptContainer = style([display(`flex), width(`px(240))]);
```
Comment (evilpeach) [inline scan/src/reusable/InfoMobileCard.re]: remove

File: scan/src/reusable/InfoMobileCard.re [hunk 5]
```diff
@@ -0,0 +1,44 @@
+type t =
+  | BlockHeight(ID.Block.t)
```
Comment (evilpeach) [inline scan/src/reusable/InfoMobileCard.re]: rename to `Height`

File: scan/src/reusable/InfoMobileCard.re [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  | BlockTXNCount(int)
```
Comment (evilpeach) [inline scan/src/reusable/InfoMobileCard.re]: rename to `Count`

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by tansawit — CHANGES_REQUESTED** [review]

**Review by tansawit — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**